patients/models.py

```diff
@@ -1,3 +1,14 @@
 from django.db import models
 
 # Create your models here.
+
+class PatientName(models.Model):

```

Comment:
Is this model about a `Patient` OR `PatientName`?
Is date_added an attribute of a `Patient` or `PatientName`